libs/designsystem/src/lib/components/modal/services/modal.helper.spec.ts

```diff
@@ -183,6 +188,15 @@ describe('ModalHelper', () => {
     });
   };
 
+  const expectBodyToAllowScroll = () => {
+    it(`body should have class 'allow-background-scroll'`, () => {
+      expect(window.document.body.classList).toContain('allow-background-scroll');
+      expect(window.document.body).toHaveComputedStyle({
+        overflow: 'visible',
+      });
+    });
+  };
+
```

Comment:
Probably better to move this test into it's own `describe` block 👍🏻

Reply:
Changed in latest commit